src/platform/user/profile/vet360/tests/components/Transaction.unit.spec.jsx

```diff
@@ -19,13 +20,21 @@ describe('<Vet360Transaction/>', () => {
   it('renders', () => {
     const component = enzyme.shallow(
       <Vet360Transaction {...props}>
-        <div>Children</div>
+        <div className="content">Children</div>
       </Vet360Transaction>,
     );
 
     expect(component.html(), 'renders children components').to.contain(
       'Children',
     );
+    expect(
```

Comment:
this is slightly redundant since we are testing that the call to `html()` above renders `'Children'` but I wanted to add this test here as proof that it works so when we use the same `component.find('div.content')` down below we know that one is working as well.